alerts/openshift-virtualization-operator/KubeVirtHyperconvergedClusterOperatorPodSchedulingMightBeImbalanced.md

```diff
@@ -0,0 +1,21 @@
+# KubeVirtHyperconvergedClusterOperatorPodSchedulingMightBeImbalanced 
+
+## Meaning 
+
+This alert fires when a node contains more than then the maximum configured 
+number of images. The problem happens because the list of images on a node
+is shortened to that configured number, 50 by default. 
+
+## Impact 
+
+Pod scheduling might be imbalanced across nodes. 
+
+## Mitigation 
+
+As a workaround, you can increase the image limit by editing the `
```

Comment:
The backtick should be on the following line so that the backticks surround "KubeletConfig". Same for "nodeStatusMaxImages".

Reply:
changed